src/rachanaGJun20/String/StringCharCounting.java

```diff
@@ -0,0 +1,51 @@
+/*1) Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2*/
+
+package rachanaGJun20.String;
+
+public class StringCharCounting {
+
+	void countCharByCategory(String inputstr) { // method will count digits,letters,alphabets(U+L),special symbols
+
+		int digitcount = 0;
+		int lowercasecount = 0;
+		int uppercasecount = 0;
+		int letterscount = 0;
+		int specialcharcount = 0;
+
+		for (int i = 0; i < inputstr.length(); i++) {
+
+			if (Character.isLetter(inputstr.charAt(i))) {
+				letterscount++;
+				if (Character.isLowerCase(inputstr.charAt(i)))
+					lowercasecount++;
```

Comment:
Here nested if not required for this letter logic. Think here. Also this variable letterscount  is not required.

Reply:
ok got it got it!